Makefile

```
@@ -1,127 +1,499 @@
+# Monitoring Commands
+# =============================================
+
+.PHONY: monitor
+monitor: ## Show service health and logs
+	@echo -e "$(BLUE)Service Status:$(NC)"
+	$(MAKE) ps
+	@echo ""
+	@echo -e "$(BLUE)Recent Logs:$(NC)"
+	$(DOCKER_COMPOSE) -f $(DOCKER_COMPOSE_FILE) logs --tail=20
+
+.PHONY: health
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)